### PR TITLE
feat: [#186896489] pre-populate license task check status business info

### PR DESCRIPTION
**Pull request by AnJuHyppolite**
### Ticket

[#186896489](https://www.pivotaltracker.com/story/show/186896489)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/src/components/tasks/LicenseTask.test.tsx [hunk 1]
```diff
@@ -132,7 +138,7 @@ describe("<LicenseTask />", () => {
     });
   });
 
-  describe("form tab", () => {
+  describe("check status tab", () => {
```
Comment (AnJuHyppolite) [inline web/src/components/tasks/LicenseTask.test.tsx:141]: I updated the `describe` titles to match the copy in the tabs in the app.

**Review by cbrunefearless — APPROVED** [review]
lgtm

File: web/src/components/tasks/CheckStatus.tsx [hunk 2]
```diff
@@ -48,7 +48,17 @@ export const CheckStatus = (props: Props): ReactElement => {
   useMountEffectWhenDefined(() => {
     if (!business) return;
 
-    if (business.licenseData) {
+    if (business.formationData.formationResponse?.success) {
```
Comment (somabadri) [inline web/src/components/tasks/CheckStatus.tsx:51]: [boulder] Please correct me if I'm misunderstanding the logic flow, but based on how these `if` statements are structured, it appears the component will always render with the `formationData`. So, if a user were to, for instance, edit the pre-filled information, check their status, and then click `edit` again, wouldn't their newly inputted information be overwritten by the data from formation? Shouldn't we first check whether they have existing information in the `licenseData` before auto-filling from `formationData`? If I recall correctly from our discussion during the planning session, we agreed that `licenseData` should be the source of truth for this feature.

Comment (AnJuHyppolite) [reply web/src/components/tasks/CheckStatus.tsx:51]: Thanks for the feedback/reminder!
Updated

**Pull request merged**